src/power_grid_model/validation/rules.py

```diff
@@ -316,7 +319,7 @@ def outside(val: np.ndarray, *ref: np.ndarray) -> np.ndarray:
         return np.logical_or(np.less(val, np.minimum(*ref)), np.greater(val, np.maximum(*ref)))
 
     return none_match_comparison(
-        data, component, field, outside, (ref_value_1, ref_value_2), NotBetweenOrAtError, default_value
+        data, component, field, outside, (ref_value_1, ref_value_2), NotBetweenOrAtError, default_value, default_value_2
```

Comment:
Is there a deeper reason the parameters aren't renamed accordingly? Here and everywhere else necessary ofc.
```suggestion
        data, component, field, outside, (ref_value_1, ref_value_2), NotBetweenOrAtError, default_value_1, default_value_2
```

Reply:
@Jerry-Jinfeng-Guo no particular reason. made it default 1/2